public/js/services/userService.js

```diff
@@ -304,29 +304,34 @@ angular.module('BibBox').service('userService', ['$q', '$timeout', '$location',
     /**
      * Check FBS is online.
      *
-     * It's plased on user service as FBS has to due with users.
+     * It's placed on user service as FBS has to due with users.
```

Comment:
"This function is placed on user service as all FBS calls have to do with users."